<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>611</width>
      <height>381</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>DashBoard</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Apps and Others</string>
     </attribute>
     <widget class="QPushButton" name="app_other_open">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>OPEN</string>
      </property>
     </widget>
     <widget class="QLabel" name="app_selected">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>190</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Selected</string>
      </property>
     </widget>
     <widget class="QLabel" name="apps_other_selected_number">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>210</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="app_other_add">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QListWidget" name="apps_other_list">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>451</width>
        <height>301</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="movement">
       <enum>QListView::Free</enum>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="app_other_remove">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>REMOVE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="app_other_remove_all">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>REMOVE ALL</string>
      </property>
     </widget>
     <widget class="QLabel" name="apps_other_status">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>326</y>
        <width>451</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>History</string>
     </attribute>
     <widget class="QPushButton" name="recents_open">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>OPEN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="recents_add">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QLabel" name="recents_selected">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>190</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Selected</string>
      </property>
     </widget>
     <widget class="QLabel" name="recents_selected_number">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>210</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="recents_selectall">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SELECT ALL</string>
      </property>
     </widget>
     <widget class="QListWidget" name="recents_list">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>451</width>
        <height>301</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="movement">
       <enum>QListView::Free</enum>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="app_other_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apps and Other</string>
    </property>
   </widget>
   <widget class="QPushButton" name="historybutton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>History</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TASKEY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runningbutton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DashBoard</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
